### DISCUSSION TITLE
Dynamic "favourite beatmaps" collection

### DISCUSSION THREAD

**Discussion by spacekotek** [ppy/osu#30765] [Ideas] [answered]
The idea is to have a self-updating collection in game that contains all the beatmaps that are favourited and available.
With the addition of favourite button to the results screen, it would make sense to me to have easy access to my favourite beatmaps.
I of course can just open my profile and jump to a beatmap from there, but it loads painfully slow and there is no search.
And i also can just manually add a beatmap to a normal collection, but then whats the point of having the favourite button on the results screen.
Another thing is to have a button to download all favourited beatmaps

**Comment by Joehuu** [comment, marked as answer]
This is a duplicate of https://github.com/ppy/osu/discussions/22798.

> I of course can just open my profile and jump to a beatmap from there, but it loads painfully slow and there is no search.

There is a favorites category filter in beatmap listing that you can use for searching for now.
